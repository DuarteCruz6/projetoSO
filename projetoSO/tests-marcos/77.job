SHOW
WRITE [(i4,91)(z0,5)(b2,7)(r3,1)]
DELETE [r3,z0,i4,b2]
WRITE [(i2,10)(x1,33)(x0,18)]
WRITE [(v4,94)(d2,14)(l1,85)]
DELETE [x0,d2,v4,l1]
WRITE [(l2,84)(t4,31)(c4,95)(m6,37)]
DELETE [c4,i2,x1,l2]
DELETE [t4]
WRITE [(t7,74)(d7,69)(i3,57)(c8,6)]
READ [d7]
WRITE [(h3,49)(q8,41)(j2,25)]
DELETE [m6,h3,j2,c8,q8]
WRITE [(w0,21)(e2,18)(e1,76)(m5,6)(b7,19)]
WRITE [(n1,35)]
WRITE [(y5,67)(b8,54)(v8,95)(x0,69)]
WRITE [(j3,37)(h5,86)(o6,26)(k7,85)]
DELETE [e2,v8]
WRITE [(z7,31)(z8,82)(f2,89)]
WRITE [(x9,24)]
READ [o6]
READ [w0]
READ [f2]
DELETE [w0,b7,f2]
WRITE [(r8,44)(r6,15)(j5,61)]
WAIT 1000
READ [z7]
WRITE [(z3,49)(e9,96)(b9,88)(a4,94)]
READ [j5]
WRITE [(b3,75)(m1,83)(i7,5)(o0,27)(h7,13)]
DELETE [z8,h5,z3,y5,r8]
WRITE [(m2,30)]
WRITE [(y1,76)(h9,84)(s8,48)(a1,98)(f0,4)]
READ [h9]
READ [o0]
WRITE [(k5,16)(b4,54)(y6,11)(h6,97)(o1,62)]
WRITE [(z6,82)(t0,47)(g3,62)(f6,50)(q8,78)]
WRITE [(u6,77)]
WRITE [(r0,76)]
WRITE [(e2,97)(y8,66)]
WRITE [(e3,43)(p1,64)(h3,90)]
WRITE [(k1,43)(q6,56)(d5,71)]
READ [h7]
WRITE [(v9,72)]
WAIT 1000
WRITE [(r1,39)(z1,94)]
WRITE [(h8,48)(x8,41)]
WRITE [(n5,61)(k8,16)]
WRITE [(u2,97)(w3,29)(v4,97)(w3,17)(t2,95)]
DELETE [k5,a4,k7,e3]
WRITE [(x3,76)(k9,93)(m4,22)(y5,46)]
BACKUP
READ [x0]
WRITE [(c7,49)(r7,35)(g1,73)(q3,88)(v6,30)]
READ [t2]
