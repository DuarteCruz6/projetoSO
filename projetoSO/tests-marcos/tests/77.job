BACKUP
WAIT 1000
SHOW
WRITE [(h0,99)(y5,29)]
WRITE [(y6,0)(f8,45)(g4,90)(x6,70)]
WRITE [(c6,0)(s4,92)(g2,16)(w9,38)]
WRITE [(n6,86)(u1,2)(l5,3)(b0,22)]
READ [y5]
WRITE [(r2,90)(n8,13)(r8,97)]
WRITE [(p5,71)(r3,21)(c3,90)(n7,95)(y3,93)]
WRITE [(i9,34)(j1,63)(a6,97)(v8,58)(i0,14)]
READ [y3]
WRITE [(o2,38)]
WRITE [(f7,36)]
BACKUP
READ [f7]
WRITE [(s0,70)(h3,53)(a0,81)]
DELETE [n8,n7,r2]
READ [s4]
WRITE [(m7,76)(j6,21)]
WRITE [(h4,41)]
WRITE [(f1,40)(a5,44)(e3,33)(l1,19)]
READ [b0]
READ [l1]
READ [v8]
SHOW
READ [v8]
DELETE [l5,l1]
DELETE [a0,f7]
WRITE [(g6,96)(d8,43)(b3,69)(f3,90)(k6,9)]
WRITE [(u3,56)(o6,59)(g7,55)]
WRITE [(f0,19)(p3,90)(l9,44)(l2,89)(x9,42)]
DELETE [s0,y3,x9,l9]
SHOW
WRITE [(s5,60)(f6,26)(h7,6)(d7,99)(u5,70)]
WRITE [(e2,72)]
WRITE [(n9,23)(b6,18)(d6,44)(r0,13)(w3,65)]
WRITE [(e9,56)(j8,56)(j4,76)]
