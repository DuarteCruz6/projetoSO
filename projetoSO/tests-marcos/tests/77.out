[(y5,29)]
[(y3,93)]
[(f7,36)]
[(s4,92)]
[(b0,22)]
[(l1,19)]
[(v8,58)]
(a5, 44)
(a0, 81)
(a6, 97)
(b0, 22)
(c3, 90)
(c6, 0)
(e3, 33)
(f1, 40)
(f7, 36)
(f8, 45)
(g2, 16)
(g4, 90)
(h4, 41)
(h3, 53)
(h0, 99)
(i0, 14)
(i9, 34)
(j6, 21)
(j1, 63)
(l1, 19)
(l5, 3)
(m7, 76)
(n6, 86)
(o2, 38)
(p5, 71)
(r3, 21)
(r8, 97)
(s0, 70)
(s4, 92)
(u1, 2)
(v8, 58)
(w9, 38)
(x6, 70)
(y3, 93)
(y6, 0)
(y5, 29)
[(v8,58)]
(a5, 44)
(a6, 97)
(b3, 69)
(b0, 22)
(c3, 90)
(c6, 0)
(d8, 43)
(e3, 33)
(f0, 19)
(f3, 90)
(f1, 40)
(f8, 45)
(g7, 55)
(g6, 96)
(g2, 16)
(g4, 90)
(h4, 41)
(h3, 53)
(h0, 99)
(i0, 14)
(i9, 34)
(j6, 21)
(j1, 63)
(k6, 9)
(l2, 89)
(m7, 76)
(n6, 86)
(o6, 59)
(o2, 38)
(p3, 90)
(p5, 71)
(r3, 21)
(r8, 97)
(s4, 92)
(u3, 56)
(u1, 2)
(v8, 58)
(w9, 38)
(x6, 70)
(y6, 0)
(y5, 29)
